packages/calling/src/CallingClient/calling/call.test.ts

```diff
@@ -854,6 +888,90 @@ describe('State Machine handler tests', () => {
     expect(call['callStateMachine'].state.value).toBe('S_SEND_CALL_DISCONNECT');
   });
 
+  it('outgoing call where we receive connect directly after setup. Media established before connect. test call and media state changes', async () => {
+    const statusPayload = <WebexRequestPayload>(<unknown>{
+      statusCode: 200,
+      body: mockStatusBody,
+    });
+
+    const dummySetupEvent = {
+      type: 'E_SEND_CALL_SETUP',
+      data: undefined as any,
+    };
+
+    const dummyConnectEvent = {
+      type: 'E_RECV_CALL_CONNECT',
+      data: undefined as any,
+    };
+
+    const dummyOfferEvent = {
+      type: 'E_SEND_ROAP_OFFER',
+      data: undefined as any,
+    };
+
+    const dummyAnswerEvent = {
+      type: 'E_RECV_ROAP_ANSWER',
+      data: {
+        seq: 1,
+        messageType: 'ANSWER',
+        sdp: 'sdp',
+      },
+    };
+
+    const dummyOkEvent = {
+      type: 'E_ROAP_OK',
+      data: {
+        received: false,
+        message: {
+          seq: 1,
+          messageType: 'OK',
+        },
+      },
+    };
+
+    const postMediaSpy = jest.spyOn(call as any, 'postMedia');
+
+    webex.request.mockReturnValue(statusPayload);
+
+    call.sendCallStateMachineEvt(dummySetupEvent as CallEvent);
+    expect(call['callStateMachine'].state.value).toBe('S_SEND_CALL_SETUP');
+
+    // dummyOfferEvent.type = 'E_SEND_ROAP_OFFER';
+    call.sendMediaStateMachineEvt(dummyOfferEvent as RoapEvent);
+
+    /**
+     * Since the event doesn't have any data above, we should request media sdk for an offer here.
+     * The below event is expected to be called again my mediaSdk.
+     */
+    dummyOfferEvent.data = {
+      seq: 1,
+      messageType: 'OFFER',
+      sdp: 'sdp',
+    };
+    call.sendMediaStateMachineEvt(dummyOfferEvent as RoapEvent);
+    expect(mediaConnection.initiateOffer).toHaveBeenCalledTimes(1);
+    expect(postMediaSpy).toHaveBeenLastCalledWith(dummyOfferEvent.data as RoapMessage);
+
+    // dummyAEvent.type = 'E_RECV_ROAP_ANSWER';
```

Comment:
could you please remove this commented code as well ?

Reply:
Removed the commented code. Thanks